Sources/MapboxNavigation/LanesView.swift

```diff
@@ -82,7 +82,7 @@ open class LanesView: UIView, NavigationComponent {
         }
         
         let subviews = tertiaryInstruction.components.compactMap { (component) -> LaneView? in
-            if case let .lane(indications: indications, isUsable: isUsable) = component {
+            if case let .lane(indications: indications, isUsable: isUsable, preferredDirection: _) = component {
                 let maneuverDirection = visualInstruction?.primaryInstruction.maneuverDirection ?? ManeuverDirection(rawValue: indications.description)
```

Comment:
We’ll make use of the new `preferredDirection` associated value when we implement #2801. It would be obvious to stick this value in front of the primary maneuver direction, but `preferredDirection` is a `LaneIndication`, which isn’t as straightforward to convert to a `ManeuverDirection`. mapbox/mapbox-directions-swift#534 would make it a `ManeuverDirection`, which will greatly simplify #2801.